src/utils/utils.js

```diff
@@ -1,17 +1,16 @@
-const { db } = require("./admin");
-
-const updateDocument = (collection, document, field) => {
-
-};
+const { db, writeBatch } = require("./admin");
 
 const updateApplicationStatus = (collection, applicants) => {
-    
+    applicants.forEach((app) => {
+        writeBatch.set(db.doc(`${collection}/${app.applicant}`), { status: app.status });
+    });
```

Comment:
I think you'll need to do a set with merge. You can look it up [here](https://firebase.google.com/docs/firestore/manage-data/add-data#web-version-8_1). This will ensure that we aren't rewriting the entire doc, just merging data.